controllers/storagecluster/cephcluster.go

```diff
@@ -116,6 +117,11 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return err
 	}
 
+	if r.platform.platform == v1.IBMCloudPlatformType || r.platform.platform == IBMCloudCosPlatformType {
```

Comment:
Don't reference the variable directly, use the GetPlatform() function.

Reply:
Done